go/vt/mysqlctl/xtrabackupengine.go

```diff
@@ -269,19 +270,18 @@ func (be *XtrabackupEngine) backupFiles(ctx context.Context, params BackupParams
 
 	destWriters := []io.Writer{}
 	destBuffers := []*bufio.Writer{}
-	destCompressors := []*pgzip.Writer{}
+	destCompressors := []io.WriteCloser{}
 	for _, file := range destFiles {
 		buffer := bufio.NewWriterSize(file, writerBufferSize)
 		destBuffers = append(destBuffers, buffer)
 		writer := io.Writer(buffer)
 
 		// Create the gzip compression pipe, if necessary.
 		if *backupStorageCompress {
-			compressor, err := pgzip.NewWriterLevel(writer, pgzip.BestSpeed)
-			if err != nil {
-				return replicationPosition, vterrors.Wrap(err, "cannot create gzip compressor")
-			}
-			compressor.SetConcurrency(*backupCompressBlockSize, *backupCompressBlocks)
+			compressor := pargzip.NewWriter(writer)
+			compressor.ChunkSize = *backupCompressBlockSize
+			compressor.Parallel = *backupCompressBlocks
+			compressor.CompressionLevel = pargzip.BestCompression
```

Comment:
Did you mean BestSpeed?

Reply:
yes, pushed new version. thanks for the catch.